<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>611</width>
      <height>431</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>111</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tải bất kỳ tệp tin nào</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>81</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="photo.qrc">:/icons/abc.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>121</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tải video trên youtube</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>81</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="photo.qrc">:/icons/youtube.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>230</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>111</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Thay đổi theme và cài đặt</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>91</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="photo.qrc">:/icons/fix.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>111</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tải danh sách video</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>81</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="photo.qrc">:/icons/playlist.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>371</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>nhập đường dẫn url</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>291</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>chọn nơi lưu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>icons/search.png</normaloff>icons/search.png</iconset>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>186</y>
        <width>351</width>
        <height>16</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>240</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>bắt đầu tải</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>561</width>
        <height>451</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>1 video</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>361</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Nhập đường dẫn của youtube</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>361</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Chọn thư mục lưu</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>60</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/check.png</normaloff>icons/check.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>21</width>
          <height>21</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>100</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/save.png</normaloff>icons/save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>310</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>bắt đầu tải</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>200</y>
          <width>411</width>
          <height>16</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>160</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>chất lượng video</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>160</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>250</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>nhiều video</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>50</y>
          <width>411</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
         </font>
        </property>
        <property name="placeholderText">
         <string>nhập url abum</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>90</y>
          <width>361</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
         </font>
        </property>
        <property name="placeholderText">
         <string>Nơi lưu</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>90</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>icons/save.png</normaloff>icons/save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>150</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>chất lượng video</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_2">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>150</y>
          <width>211</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>1080</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>720</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>480</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>360</string>
         </property>
        </item>
       </widget>
       <widget class="QProgressBar" name="progressBar_3">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>270</y>
          <width>371</width>
          <height>13</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>310</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>bắt đầu tải</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>220</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>video hiện tại:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>210</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>tất cả trong danh sách video</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>210</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(255, 0, 0)</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumber_2">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>210</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 8pt &quot;Segoe UI&quot;;color: rgb(0, 255, 0)</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>270</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>161</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(50, 50, 50);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>260</y>
        <width>161</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(240, 240, 240);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>260</y>
        <width>161</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(48, 47, 47);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>461</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Chọn theme bạn muốn sử dụng</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>161</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(25, 35, 45);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>130</y>
        <width>161</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(50, 50, 50);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>97</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="photo.qrc">
      <normaloff>:/icons/browser.png</normaloff>:/icons/browser.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>131</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>131</width>
      <height>97</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="photo.qrc">
      <normaloff>:/icons/file.png</normaloff>:/icons/file.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>131</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="photo.qrc">
      <normaloff>:/icons/setting.png</normaloff>:/icons/setting.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>131</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>131</width>
      <height>97</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="photo.qrc">
      <normaloff>:/icons/youtube.png</normaloff>:/icons/youtube.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>131</width>
      <height>80</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="photo.qrc"/>
 </resources>
 <connections/>
</ui>
